<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSettings</class>
 <widget class="QDialog" name="PanelSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Panel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Applets</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="generalBox">
           <property name="title">
            <string>General Settings</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="iconThemeLabel">
              <property name="text">
               <string>Icon theme:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="theme"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="panelBox">
           <property name="title">
            <string>Panel Settings</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="verticalPositionLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vertical position:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="verticalPosition">
              <item>
               <property name="text">
                <string>Top</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bottom</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="fontLabel">
              <property name="text">
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QFontComboBox" name="font">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>screen:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="screen"/>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="fontSize"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>57</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QListWidget" name="appletsList"/>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QToolButton" name="appletUp">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="go-up">
               <normaloff/>
              </iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="appletDown">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="go-down">
               <normaloff/>
              </iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>6</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="appletAdd">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="list-add">
               <normaloff/>
              </iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="appletRemove">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff/>
              </iconset>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>6</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="appletSettings">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="icon">
              <iconset theme="preferences-system">
               <normaloff/>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Note: Changes made in this page cannot be reset.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PanelSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
